CosmosContracts/juno-tools
# refactor(pages): rework airdrops create page

Fixes #57

## Description

This week-long PR updates the create airdrop page according to the Figma spec.

## Changes

- [x] match create airdrop page designs with figma
- [x] extract form control into separate component
- [x] ~~rework form logic to react-hook-form~~ (separate PR)
- [x] ~~exhaustive form component splitting~~ (separate PR)

## Screenshots

https://cdn.discordapp.com/attachments/933515307597848656/947554886529196072/junotools-create-airdrop-page-2022-02-28.gif

![](https://cdn.discordapp.com/attachments/933515307597848656/947554886529196072/junotools-create-airdrop-page-2022-02-28.gif)

## Testing Steps

As a reviewer, what steps should I take to verify this is working correctly?

- ensure all inputs are fillable and clickable
- try both click to upload csv and drag file to input area

## Links

\-
